src/GraphQL.Serialization.Newtonsoft/NewtonsoftContractResolver.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Reflection;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace GraphQL.Serialization.Newtonsoft
+{
+    internal class NewtonsoftContractResolver : DefaultContractResolver
```

Comment:
Consider to make this public in case when someone want to tune this resolver.

Reply:
Done